src/inference_profiling/concurrent_inference.py

```diff
@@ -0,0 +1,56 @@
+# The given script sends in multiple http requests at a particular time which can be used to test
+# the performance of the torchserve deployed on Cloud run or as a standalone docker container.
+
+# For 10-100 requests, the container memory capacity on cloud run remains upto 50%.
+# CPU performance is around 53% for 10 requests and around 80% for 1000 requests
+# Container instance count 30 for 10 requests and around 139 for 1000 requests
+# Container startup latency 2-3s for 10-1000 requests. Anamolous reading of 5s found once.
+# Max reuest latency was found out to be 1.06 min.Anamolous reading of 7.91 minutes found once.
```

Comment:
Couple of spelling mistakes here, please correct them.

Also looks like some of the same information as src/inference_profiling/cloud_run_logs.txt (e.g. CPU performance), does it need to be deplucated?

Reply:
I have updated the cloud_run_logs.txt